openfl/transport/grpc/aggregator_client.py

```diff
@@ -148,15 +148,15 @@ def wrapper(self, *args, **kwargs):
         while True:
             try:
                 response = func(self, *args, **kwargs)
+                break
```

Comment:
I guess this replaces the `continue` statement from the deleted code block below?